lldb/tools/lldb-dap/LLDBUtils.h

```diff
@@ -33,9 +39,14 @@ namespace lldb_dap {
 /// \param[in] strm
 ///     The stream that will receive the prefix, prompt + command and
 ///     all command output.
+///
+/// \param[in] parse_command_directives
+///     If \b false, then command prefixes like \b ! or \b ? are not parsed and
+///     each command is executed verbatim.
 void RunLLDBCommands(llvm::StringRef prefix,
```

Comment:
return a `std::string` that will be the first error for any `!` prefixed lines that fail, and return empty string if no commands started with `!` or some did but none failed? Then the callers of this function can decide what to do with it.